change/@fluentui-react-charting-cd6618b3-30a0-474b-b2e9-9f008ff45c69.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1045 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26861&commitId=3aa2e0581e63a75e6743e9a1c545defe49410faa&env=prod)

✅ There was 1045 screenshots added, 0 screenshots removed, 0 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>OverflowSet.Root - RTL.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet.Root - RTL.default.chromium_1.png|0| Added|

</details>


  <details><summary><b>OverflowSet.Root.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet.Root.default.chromium_1.png|0| Added|

</details>


  <details><summary><b>SearchBox.Full - RTL.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.Full - RTL.default.chromium_1.png|0| Added|

</details>


  <details><summary><b>SearchBox.Full.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.Full.default.chromium_1.png|0| Added|

</details>


  <details><summary><b>SearchBox.Root - RTL.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.Root - RTL.default.chromium_1.png|0| Added|

</details>


  <details><summary><b>SearchBox.Root.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.Root.default.chromium_1.png|0| Added|

</details>


  <details><summary><b>SearchBox.ShowIcon - RTL.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.ShowIcon - RTL.default.chromium_1.png|0| Added|

</details>


  <details><summary><b>SearchBox.ShowIcon.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.ShowIcon.default.chromium_1.png|0| Added|

</details>


  <details><summary><b>unknown</b> 1037 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over a wide suggestion element.chromium.png|0| Added|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over the X button on a wide suggestion element.chromium.png|0| Added|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.default.chromium.png|0| Added|
|ActivityItem.Compact.default.chromium.png|0| Added|
|ActivityItem.CompactPersonas.default.chromium.png|0| Added|
|ActivityItem.Personas - RTL.default.chromium.png|0| Added|
|ActivityItem.Personas.default.chromium.png|0| Added|
|ActivityItem.Root - RTL.default.chromium.png|0| Added|
|ActivityItem.Root.default.chromium.png|0| Added|
|Breadcrumb.Button - RTL.click.chromium.png|0| Added|
|Breadcrumb.Button - RTL.default.chromium.png|0| Added|
|Breadcrumb.Button - RTL.hover.chromium.png|0| Added|
|Breadcrumb.Button - RTL.itemLinkFocus.chromium.png|0| Added|
|Breadcrumb.Button - RTL.longTitleHover.chromium.png|0| Added|
|Breadcrumb.Button - RTL.overflowButtonFocus.chromium.png|0| Added|
|Breadcrumb.Button - RTL.shortTitleHover.chromium.png|0| Added|
|Breadcrumb.Button.click.chromium.png|0| Added|
|Breadcrumb.Button.default.chromium.png|0| Added|
|Breadcrumb.Button.hover.chromium.png|0| Added|
|Breadcrumb.Button.itemLinkFocus.chromium.png|0| Added|
|Breadcrumb.Button.longTitleHover.chromium.png|0| Added|
|Breadcrumb.Button.overflowButtonFocus.chromium.png|0| Added|
|Breadcrumb.Button.shortTitleHover.chromium.png|0| Added|
|Breadcrumb.Hovering items.actionable hover.chromium.png|0| Added|
|Breadcrumb.Hovering items.actionable overflow hover.chromium.png|0| Added|
|Breadcrumb.Hovering items.chromium.png|0| Added|
|Breadcrumb.Hovering items.non-actionable hover.chromium.png|0| Added|
|Breadcrumb.Hovering items.non-actionable overflow hover.chromium.png|0| Added|
|Breadcrumb.Root - RTL.click.chromium.png|0| Added|
|Breadcrumb.Root - RTL.default.chromium.png|0| Added|
|Breadcrumb.Root - RTL.hover.chromium.png|0| Added|
|Breadcrumb.Root - RTL.itemLinkFocus.chromium.png|0| Added|
|Breadcrumb.Root - RTL.longTitleHover.chromium.png|0| Added|
|Breadcrumb.Root - RTL.overflowButtonFocus.chromium.png|0| Added|
|Breadcrumb.Root - RTL.shortTitleHover.chromium.png|0| Added|
|Breadcrumb.Root.click.chromium.png|0| Added|
|Breadcrumb.Root.default.chromium.png|0| Added|
|Breadcrumb.Root.hover.chromium.png|0| Added|
|Breadcrumb.Root.itemLinkFocus.chromium.png|0| Added|
|Breadcrumb.Root.longTitleHover.chromium.png|0| Added|
|Breadcrumb.Root.overflowButtonFocus.chromium.png|0| Added|
|Breadcrumb.Root.shortTitleHover.chromium.png|0| Added|
|Button (compat).Checked.default.chromium.png|0| Added|
|Button (compat).Checked.hover.chromium.png|0| Added|
|Button (compat).Checked.pressed.chromium.png|0| Added|
|Button (compat).Disabled.default.chromium.png|0| Added|
|Button (compat).Disabled.hover.chromium.png|0| Added|
|Button (compat).Disabled.pressed.chromium.png|0| Added|
|Button (compat).Icon Only.default.chromium.png|0| Added|
|Button (compat).Icon Only.hover.chromium.png|0| Added|
|Button (compat).Icon Only.pressed.chromium.png|0| Added|
|Button (compat).No Icon.default.chromium.png|0| Added|
|Button (compat).No Icon.hover.chromium.png|0| Added|
|Button (compat).No Icon.pressed.chromium.png|0| Added|
|Button (compat).Primary Anchor.default.chromium.png|0| Added|
|Button (compat).Primary Anchor.hover.chromium.png|0| Added|
|Button (compat).Primary Anchor.pressed.chromium.png|0| Added|
|Button (compat).Primary Checked.default.chromium.png|0| Added|
|Button (compat).Primary Checked.hover.chromium.png|0| Added|
|Button (compat).Primary Checked.pressed.chromium.png|0| Added|
|Button (compat).Primary Disabled.default.chromium.png|0| Added|
|Button (compat).Primary Disabled.hover.chromium.png|0| Added|
|Button (compat).Primary Disabled.pressed.chromium.png|0| Added|
|Button (compat).Primary.default.chromium.png|0| Added|
|Button (compat).Primary.hover.chromium.png|0| Added|
|Button (compat).Primary.pressed.chromium.png|0| Added|
|Button (compat).Root - RTL.default.chromium.png|0| Added|
|Button (compat).Root - RTL.hover.chromium.png|0| Added|
|Button (compat).Root - RTL.pressed.chromium.png|0| Added|
|Button (compat).Root.default.chromium.png|0| Added|
|Button (compat).Root.hover.chromium.png|0| Added|
|Button (compat).Root.pressed.chromium.png|0| Added|
|Button Action (compat).Checked.default.chromium.png|0| Added|
|Button Action (compat).Checked.hover.chromium.png|0| Added|
|Button Action (compat).Checked.pressed.chromium.png|0| Added|
|Button Action (compat).Disabled.default.chromium.png|0| Added|
|Button Action (compat).Disabled.hover.chromium.png|0| Added|
|Button Action (compat).Disabled.pressed.chromium.png|0| Added|
|Button Action (compat).Icon Only.default.chromium.png|0| Added|
|Button Action (compat).Icon Only.hover.chromium.png|0| Added|
|Button Action (compat).Icon Only.pressed.chromium.png|0| Added|
|Button Action (compat).No Icon.default.chromium.png|0| Added|
|Button Action (compat).No Icon.hover.chromium.png|0| Added|
|Button Action (compat).No Icon.pressed.chromium.png|0| Added|
|Button Action (compat).Root - RTL.default.chromium.png|0| Added|
|Button Action (compat).Root - RTL.hover.chromium.png|0| Added|
|Button Action (compat).Root - RTL.pressed.chromium.png|0| Added|
|Button Action (compat).Root.default.chromium.png|0| Added|
|Button Action (compat).Root.hover.chromium.png|0| Added|
|Button Action (compat).Root.pressed.chromium.png|0| Added|
|Button Command (compat).Checked.default.chromium.png|0| Added|
|Button Command (compat).Checked.hover.chromium.png|0| Added|
|Button Command (compat).Checked.open.chromium.png|0| Added|
|Button Command (compat).Checked.pressed.chromium.png|0| Added|
|Button Command (compat).Disabled.default.chromium.png|0| Added|
|Button Command (compat).Disabled.hover.chromium.png|0| Added|
|Button Command (compat).Disabled.open.chromium.png|0| Added|
|Button Command (compat).Disabled.pressed.chromium.png|0| Added|
|Button Command (compat).Root - RTL.default.chromium.png|0| Added|
|Button Command (compat).Root - RTL.hover.chromium.png|0| Added|
|Button Command (compat).Root - RTL.open.chromium.png|0| Added|
|Button Command (compat).Root - RTL.pressed.chromium.png|0| Added|
|Button Command (compat).Root.default.chromium.png|0| Added|
|Button Command (compat).Root.hover.chromium.png|0| Added|
|Button Command (compat).Root.open.chromium.png|0| Added|
|Button Command (compat).Root.pressed.chromium.png|0| Added|
|Button Compound (compat).Checked.default.chromium.png|0| Added|
|Button Compound (compat).Checked.hover.chromium.png|0| Added|
|Button Compound (compat).Checked.pressed.chromium.png|0| Added|
|Button Compound (compat).Disabled.default.chromium.png|0| Added|
|Button Compound (compat).Disabled.hover.chromium.png|0| Added|
|Button Compound (compat).Disabled.pressed.chromium.png|0| Added|
|Button Compound (compat).No Icon.default.chromium.png|0| Added|
|Button Compound (compat).No Icon.hover.chromium.png|0| Added|
|Button Compound (compat).No Icon.pressed.chromium.png|0| Added|
|Button Compound (compat).Primary Checked.default.chromium.png|0| Added|
|Button Compound (compat).Primary Checked.hover.chromium.png|0| Added|
|Button Compound (compat).Primary Checked.pressed.chromium.png|0| Added|
|Button Compound (compat).Primary Disabled.default.chromium.png|0| Added|
|Button Compound (compat).Primary Disabled.hover.chromium.png|0| Added|
|Button Compound (compat).Primary Disabled.pressed.chromium.png|0| Added|
|Button Compound (compat).Primary.default.chromium.png|0| Added|
|Button Compound (compat).Primary.hover.chromium.png|0| Added|
|Button Compound (compat).Primary.pressed.chromium.png|0| Added|
|Button Compound (compat).Root - RTL.default.chromium.png|0| Added|
|Button Compound (compat).Root - RTL.hover.chromium.png|0| Added|
|Button Compound (compat).Root - RTL.pressed.chromium.png|0| Added|
|Button Compound (compat).Root.default.chromium.png|0| Added|
|Button Compound (compat).Root.hover.chromium.png|0| Added|
|Button Compound (compat).Root.pressed.chromium.png|0| Added|
|Button Special Scenarios (compat).no flex shrink.default.chromium.png|0| Added|
|Button Special Scenarios (compat).primary with placeholder.default.chromium.png|0| Added|
|Button Split (compat).Checked.default.chromium.png|0| Added|
|Button Split (compat).Checked.hover main.chromium.png|0| Added|
|Button Split (compat).Checked.hover split.chromium.png|0| Added|
|Button Split (compat).Command Split.default.chromium.png|0| Added|
|Button Split (compat).Command Split.hover main.chromium.png|0| Added|
|Button Split (compat).Command Split.hover split.chromium.png|0| Added|
|Button Split (compat).Default with Primary Action Disabled.default.chromium.png|0| Added|
|Button Split (compat).Default with Primary Action Disabled.hover main.chromium.png|0| Added|
|Button Split (compat).Default with Primary Action Disabled.hover split.chromium.png|0| Added|
|Button Split (compat).Disabled.default.chromium.png|0| Added|
|Button Split (compat).Disabled.hover main.chromium.png|0| Added|
|Button Split (compat).Disabled.hover split.chromium.png|0| Added|
|Button Split (compat).Primary Checked.default.chromium.png|0| Added|
|Button Split (compat).Primary Checked.hover main.chromium.png|0| Added|
|Button Split (compat).Primary Checked.hover split.chromium.png|0| Added|
|Button Split (compat).Primary Disabled.default.chromium.png|0| Added|
|Button Split (compat).Primary Disabled.hover main.chromium.png|0| Added|
|Button Split (compat).Primary Disabled.hover split.chromium.png|0| Added|
|Button Split (compat).Primary with Primary Action Disabled.default.chromium.png|0| Added|
|Button Split (compat).Primary with Primary Action Disabled.hover main.chromium.png|0| Added|
|Button Split (compat).Primary with Primary Action Disabled.hover split.chromium.png|0| Added|
|Button Split (compat).Primary.default.chromium.png|0| Added|
|Button Split (compat).Primary.hover main.chromium.png|0| Added|
|Button Split (compat).Primary.hover split.chromium.png|0| Added|
|Button Split (compat).Root - RTL.default.chromium.png|0| Added|
|Button Split (compat).Root - RTL.hover main.chromium.png|0| Added|
|Button Split (compat).Root - RTL.hover split.chromium.png|0| Added|
|Button Split (compat).Root.default.chromium.png|0| Added|
|Button Split (compat).Root.hover main.chromium.png|0| Added|
|Button Split (compat).Root.hover split.chromium.png|0| Added|
|Calendar - No Month Option.Show Month as Overlay and no Go To Today.default.chromium.png|0| Added|
|Calendar - No Month Option.Show Month selection type.default.chromium.png|0| Added|
|Calendar - No Month Option.Show Week selection type.default.chromium.png|0| Added|
|Calendar - No Month Option.Show Work Week selection type.default.chromium.png|0| Added|
|Calendar.Root - RTL.default.chromium.png|0| Added|
|Calendar.Root.default.chromium.png|0| Added|
|Callout.Beak 25.default.chromium.png|0| Added|
|Callout.Bottom auto edge.default.chromium.png|0| Added|
|Callout.Bottom center.default.chromium.png|0| Added|
|Callout.Bottom left edge - RTL.default.chromium.png|0| Added|
|Callout.Bottom left edge.default.chromium.png|0| Added|
|Callout.Bottom right edge - RTL.default.chromium.png|0| Added|
|Callout.Bottom right edge.default.chromium.png|0| Added|
|Callout.Gap space 25.default.chromium.png|0| Added|
|Callout.Left bottom edge.default.chromium.png|0| Added|
|Callout.Left center.default.chromium.png|0| Added|
|Callout.Left top edge.default.chromium.png|0| Added|
|Callout.No beak.default.chromium.png|0| Added|
|Callout.No callout width specified.default.chromium.png|0| Added|
|Callout.Rendering callout attached to a rectangle.default.chromium.png|0| Added|
|Callout.Right bottom edge.default.chromium.png|0| Added|
|Callout.Right center.default.chromium.png|0| Added|
|Callout.Right top edge.default.chromium.png|0| Added|
|Callout.Root.default.chromium.png|0| Added|
|Callout.Top auto edge.default.chromium.png|0| Added|
|Callout.Top center.default.chromium.png|0| Added|
|Callout.Top left edge.default.chromium.png|0| Added|
|Callout.Top right edge.default.chromium.png|0| Added|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.clicked.chromium.png|0| Added|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.default.chromium.png|0| Added|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.hover.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.clicked.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.default.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.hover.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.clicked.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.default.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.hover.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.clicked.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.default.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.hover.chromium.png|0| Added|
|Checkbox.Checked disabled.default.chromium.png|0| Added|
|Checkbox.Checked.default.chromium.png|0| Added|
|Checkbox.Controlled Indeterminate disabled.default.chromium.png|0| Added|
|Checkbox.Controlled Indeterminate.default.chromium.png|0| Added|
|Checkbox.Custom render Checkbox.default.chromium.png|0| Added|
|Checkbox.End - RTL.default.chromium.png|0| Added|
|Checkbox.End.default.chromium.png|0| Added|
|Checkbox.Multi-line Checkbox.default.chromium.png|0| Added|
|Checkbox.Unchecked - RTL.default.chromium.png|0| Added|
|Checkbox.Unchecked disabled.default.chromium.png|0| Added|
|Checkbox.Unchecked.default.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate disabled.default.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate.default.chromium.png|0| Added|
|ChoiceGroup.Required.default.chromium.png|0| Added|
|ChoiceGroup.Required.hover selected.chromium.png|0| Added|
|ChoiceGroup.Required.hover unselected.chromium.png|0| Added|
|ChoiceGroup.Required.selected.chromium.png|0| Added|
|ChoiceGroup.Root.default.chromium.png|0| Added|
|ChoiceGroup.Root.hover selected.chromium.png|0| Added|
|ChoiceGroup.Root.hover unselected.chromium.png|0| Added|
|ChoiceGroup.Root.selected.chromium.png|0| Added|
|ChoiceGroup.With default size images.default.chromium.png|0| Added|
|ChoiceGroup.With default size images.hover selected.chromium.png|0| Added|
|ChoiceGroup.With default size images.hover unselected.chromium.png|0| Added|
|ChoiceGroup.With default size images.selected.chromium.png|0| Added|
|ChoiceGroup.With icons - RTL.default.chromium.png|0| Added|
|ChoiceGroup.With icons - RTL.hover selected.chromium.png|0| Added|
|ChoiceGroup.With icons - RTL.hover unselected.chromium.png|0| Added|
|ChoiceGroup.With icons - RTL.selected.chromium.png|0| Added|
|ChoiceGroup.With icons.default.chromium.png|0| Added|
|ChoiceGroup.With icons.hover selected.chromium.png|0| Added|
|ChoiceGroup.With icons.hover unselected.chromium.png|0| Added|
|ChoiceGroup.With icons.selected.chromium.png|0| Added|
|ChoiceGroup.With large size images.default.chromium.png|0| Added|
|ChoiceGroup.With large size images.hover selected.chromium.png|0| Added|
|ChoiceGroup.With large size images.hover unselected.chromium.png|0| Added|
|ChoiceGroup.With large size images.selected.chromium.png|0| Added|
|Coachmark.Collapsed.default.chromium.png|0| Added|
|Coachmark.Rendering Coachmark attached to a rectangle.default.chromium.png|0| Added|
|ColorPicker.Blue.default.chromium.png|0| Added|
|ColorPicker.Root - RTL.default.chromium.png|0| Added|
|ColorPicker.Root.default.chromium.png|0| Added|
|ComboBox.Disabled.click.chromium.png|0| Added|
|ComboBox.Disabled.default.chromium.png|0| Added|
|ComboBox.Disabled.hover.chromium.png|0| Added|
|ComboBox.Error.click.chromium.png|0| Added|
|ComboBox.Error.default.chromium.png|0| Added|
|ComboBox.Error.hover.chromium.png|0| Added|
|ComboBox.Placeholder.click.chromium.png|0| Added|
|ComboBox.Placeholder.default.chromium.png|0| Added|
|ComboBox.Placeholder.hover.chromium.png|0| Added|
|ComboBox.Root - RTL.click.chromium.png|0| Added|
|ComboBox.Root - RTL.default.chromium.png|0| Added|
|ComboBox.Root - RTL.hover.chromium.png|0| Added|
|ComboBox.Root.click.chromium.png|0| Added|
|ComboBox.Root.default.chromium.png|0| Added|
|ComboBox.Root.hover.chromium.png|0| Added|
|ComboBox.Styled.click.chromium.png|0| Added|
|ComboBox.Styled.default.chromium.png|0| Added|
|ComboBox.Styled.hover.chromium.png|0| Added|
|CommandBar.Icons only.click.chromium.png|0| Added|
|CommandBar.Icons only.default.chromium.png|0| Added|
|CommandBar.Icons only.hover.chromium.png|0| Added|
|CommandBar.Root - RTL.click.chromium.png|0| Added|
|CommandBar.Root - RTL.default.chromium.png|0| Added|
|CommandBar.Root - RTL.hover.chromium.png|0| Added|
|CommandBar.Root.click.chromium.png|0| Added|
|CommandBar.Root.default.chromium.png|0| Added|
|CommandBar.Root.hover.chromium.png|0| Added|
|CommandBar.Text only.click.chromium.png|0| Added|
|CommandBar.Text only.default.chromium.png|0| Added|
|CommandBar.Text only.hover.chromium.png|0| Added|
|ContextualMenu.Root.click.chromium.png|0| Added|
|ContextualMenu.Root.default.chromium.png|0| Added|
|ContextualMenu.Root.hover.chromium.png|0| Added|
|ContextualMenu.With headers.click.chromium.png|0| Added|
|ContextualMenu.With headers.default.chromium.png|0| Added|
|ContextualMenu.With headers.hover.chromium.png|0| Added|
|ContextualMenu.With icons.click.chromium.png|0| Added|
|ContextualMenu.With icons.default.chromium.png|0| Added|
|ContextualMenu.With icons.hover.chromium.png|0| Added|
|ContextualMenu.With secondaryText - RTL.click.chromium.png|0| Added|
|ContextualMenu.With secondaryText - RTL.default.chromium.png|0| Added|
|ContextualMenu.With secondaryText - RTL.hover.chromium.png|0| Added|
|ContextualMenu.With secondaryText.click.chromium.png|0| Added|
|ContextualMenu.With secondaryText.default.chromium.png|0| Added|
|ContextualMenu.With secondaryText.hover.chromium.png|0| Added|
|ContextualMenu.With split button submenu.click.chromium.png|0| Added|
|ContextualMenu.With split button submenu.default.chromium.png|0| Added|
|ContextualMenu.With split button submenu.hover.chromium.png|0| Added|
|ContextualMenu.With submenu - RTL.click.chromium.png|0| Added|
|ContextualMenu.With submenu - RTL.default.chromium.png|0| Added|
|ContextualMenu.With submenu - RTL.hover.chromium.png|0| Added|
|ContextualMenu.With submenu.click.chromium.png|0| Added|
|ContextualMenu.With submenu.default.chromium.png|0| Added|
|ContextualMenu.With submenu.hover.chromium.png|0| Added|
|ContextualMenu.With submenus with hrefs.chromium.png|0| Added|
|ContextualMenu.With submenus with hrefs.item1 hovered.chromium.png|0| Added|
|ContextualMenu.With submenus with hrefs.item2 hovered.chromium.png|0| Added|
|ContextualMenu.With submenus with hrefs.menu opened.chromium.png|0| Added|
|ContextualMenu.With submenus with hrefs.parent hovered.chromium.png|0| Added|
|DatePicker - Disabled.With Label.click.chromium.png|0| Added|
|DatePicker - Disabled.With Label.default.chromium.png|0| Added|
|DatePicker - Disabled.With Label.hover datepicker.chromium.png|0| Added|
|DatePicker - Disabled.Without Label.click.chromium.png|0| Added|
|DatePicker - Disabled.Without Label.default.chromium.png|0| Added|
|DatePicker - Disabled.Without Label.hover datepicker.chromium.png|0| Added|
|DatePicker - Disabled.Without Value.click.chromium.png|0| Added|
|DatePicker - Disabled.Without Value.default.chromium.png|0| Added|
|DatePicker - Disabled.Without Value.hover datepicker.chromium.png|0| Added|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.click.chromium.png|0| Added|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.default.chromium.png|0| Added|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.hover datepicker.chromium.png|0| Added|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.hover day.chromium.png|0| Added|
|DatePicker.Allow text input.click.chromium.png|0| Added|
|DatePicker.Allow text input.default.chromium.png|0| Added|
|DatePicker.Allow text input.hover datepicker.chromium.png|0| Added|
|DatePicker.Allow text input.hover day.chromium.png|0| Added|
|DatePicker.Allow text input.hover month.chromium.png|0| Added|
|DatePicker.Placeholder.click.chromium.png|0| Added|
|DatePicker.Placeholder.default.chromium.png|0| Added|
|DatePicker.Placeholder.hover datepicker.chromium.png|0| Added|
|DatePicker.Placeholder.hover day.chromium.png|0| Added|
|DatePicker.Placeholder.hover month.chromium.png|0| Added|
|DatePicker.Required.click.chromium.png|0| Added|
|DatePicker.Required.default.chromium.png|0| Added|
|DatePicker.Required.hover datepicker.chromium.png|0| Added|
|DatePicker.Required.hover day.chromium.png|0| Added|
|DatePicker.Required.hover month.chromium.png|0| Added|
|DatePicker.Root - RTL.click.chromium.png|0| Added|
|DatePicker.Root - RTL.default.chromium.png|0| Added|
|DatePicker.Root - RTL.hover datepicker.chromium.png|0| Added|
|DatePicker.Root - RTL.hover day.chromium.png|0| Added|
|DatePicker.Root - RTL.hover month.chromium.png|0| Added|
|DatePicker.Root.click.chromium.png|0| Added|
|DatePicker.Root.default.chromium.png|0| Added|
|DatePicker.Root.hover datepicker.chromium.png|0| Added|
|DatePicker.Root.hover day.chromium.png|0| Added|
|DatePicker.Root.hover month.chromium.png|0| Added|
|DatePicker.Underlined and Required.click.chromium.png|0| Added|
|DatePicker.Underlined and Required.default.chromium.png|0| Added|
|DatePicker.Underlined and Required.hover datepicker.chromium.png|0| Added|
|DatePicker.Underlined and Required.hover day.chromium.png|0| Added|
|DatePicker.Underlined and Required.hover month.chromium.png|0| Added|
|DatePicker.Underlined.click.chromium.png|0| Added|
|DatePicker.Underlined.default.chromium.png|0| Added|
|DatePicker.Underlined.hover datepicker.chromium.png|0| Added|
|DatePicker.Underlined.hover day.chromium.png|0| Added|
|DatePicker.Underlined.hover month.chromium.png|0| Added|
|DetailsHeader.Root.borderWhileDragging.chromium.png|0| Added|
|DetailsHeader.Root.default.chromium.png|0| Added|
|DetailsHeader.Root.dropHint.chromium.png|0| Added|
|DetailsHeader.Root.hoverDraggable.chromium.png|0| Added|
|DetailsHeader.Root.hoverFrozenFirst.chromium.png|0| Added|
|DetailsHeader.Root.hoverFrozenLast.chromium.png|0| Added|
|DetailsList.Checkbox Visible Always - RTL.click.chromium.png|0| Added|
|DetailsList.Checkbox Visible Always - RTL.default.chromium.png|0| Added|
|DetailsList.Checkbox Visible Always - RTL.hover.chromium.png|0| Added|
|DetailsList.Checkbox Visible Always.click.chromium.png|0| Added|
|DetailsList.Checkbox Visible Always.default.chromium.png|0| Added|
|DetailsList.Checkbox Visible Always.hover.chromium.png|0| Added|
|DetailsList.Compact.click.chromium.png|0| Added|
|DetailsList.Compact.default.chromium.png|0| Added|
|DetailsList.Compact.hover.chromium.png|0| Added|
|DetailsList.Grouped with Checkbox Hidden.click.chromium.png|0| Added|
|DetailsList.Grouped with Checkbox Hidden.default.chromium.png|0| Added|
|DetailsList.Grouped with Checkbox Hidden.hover.chromium.png|0| Added|
|DetailsList.Grouped.click.chromium.png|0| Added|
|DetailsList.Grouped.default.chromium.png|0| Added|
|DetailsList.Grouped.hover.chromium.png|0| Added|
|DetailsList.Root.click.chromium.png|0| Added|
|DetailsList.Root.default.chromium.png|0| Added|
|DetailsList.Root.hover.chromium.png|0| Added|
|DetailsList.Single Selection Mode.click.chromium.png|0| Added|
|DetailsList.Single Selection Mode.default.chromium.png|0| Added|
|DetailsList.Single Selection Mode.hover.chromium.png|0| Added|
|Dialog.Blocking.default.chromium.png|0| Added|
|Dialog.Large header.default.chromium.png|0| Added|
|Dialog.Root - RTL.default.chromium.png|0| Added|
|Dialog.Root.default.chromium.png|0| Added|
|Dialog.Wide Dialog.default.chromium.png|0| Added|
|DocumentCard.Compact with preview image.default.chromium.png|0| Added|
|DocumentCard.Compact with preview list.default.chromium.png|0| Added|
|DocumentCard.Not truncated.default.chromium.png|0| Added|
|DocumentCard.With secondary title style.default.chromium.png|0| Added|
|Dropdown Disabled.Root.click.chromium.png|0| Added|
|Dropdown Disabled.Root.default.chromium.png|0| Added|
|Dropdown Disabled.Root.hover.chromium.png|0| Added|
|Dropdown.Custom Dropdown - RTL.click.chromium.png|0| Added|
|Dropdown.Custom Dropdown - RTL.default.chromium.png|0| Added|
|Dropdown.Custom Dropdown - RTL.hover item.chromium.png|0| Added|
|Dropdown.Custom Dropdown - RTL.hover.chromium.png|0| Added|
|Dropdown.Custom Dropdown.click.chromium.png|0| Added|
|Dropdown.Custom Dropdown.default.chromium.png|0| Added|
|Dropdown.Custom Dropdown.hover item.chromium.png|0| Added|
|Dropdown.Custom Dropdown.hover.chromium.png|0| Added|
|Dropdown.Disabled option selected.click.chromium.png|0| Added|
|Dropdown.Disabled option selected.default.chromium.png|0| Added|
|Dropdown.Disabled option selected.hover item.chromium.png|0| Added|
|Dropdown.Disabled option selected.hover.chromium.png|0| Added|
|Dropdown.Multiselect - RTL.click.chromium.png|0| Added|
|Dropdown.Multiselect - RTL.default.chromium.png|0| Added|
|Dropdown.Multiselect - RTL.hover item.chromium.png|0| Added|
|Dropdown.Multiselect - RTL.hover.chromium.png|0| Added|
|Dropdown.Multiselect.click.chromium.png|0| Added|
|Dropdown.Multiselect.default.chromium.png|0| Added|
|Dropdown.Multiselect.hover item.chromium.png|0| Added|
|Dropdown.Multiselect.hover.chromium.png|0| Added|
|Dropdown.Required without label.click.chromium.png|0| Added|
|Dropdown.Required without label.default.chromium.png|0| Added|
|Dropdown.Required without label.hover item.chromium.png|0| Added|
|Dropdown.Required without label.hover.chromium.png|0| Added|
|Dropdown.Required.click.chromium.png|0| Added|
|Dropdown.Required.default.chromium.png|0| Added|
|Dropdown.Required.hover item.chromium.png|0| Added|
|Dropdown.Required.hover.chromium.png|0| Added|
|Dropdown.Root - RTL.click.chromium.png|0| Added|
|Dropdown.Root - RTL.default.chromium.png|0| Added|
|Dropdown.Root - RTL.hover item.chromium.png|0| Added|
|Dropdown.Root - RTL.hover.chromium.png|0| Added|
|Dropdown.Root.click.chromium.png|0| Added|
|Dropdown.Root.default.chromium.png|0| Added|
|Dropdown.Root.hover item.chromium.png|0| Added|
|Dropdown.Root.hover.chromium.png|0| Added|
|Facepile.Add face.default.chromium.png|0| Added|
|Facepile.Custom button styles.default.chromium.png|0| Added|
|Facepile.Extra extra small.default.chromium.png|0| Added|
|Facepile.Overflow.default.chromium.png|0| Added|
|Facepile.Root - RTL.default.chromium.png|0| Added|
|Facepile.Root.default.chromium.png|0| Added|
|FocusTrapZones.Dialog nested in Panel - RTL.click on panel close button.chromium.png|0| Added|
|FocusTrapZones.Dialog nested in Panel - RTL.default.chromium.png|0| Added|
|FocusTrapZones.Dialog nested in Panel.click on panel close button.chromium.png|0| Added|
|FocusTrapZones.Dialog nested in Panel.default.chromium.png|0| Added|
|FocusTrapZones.Panel on its own.click on panel close button.chromium.png|0| Added|
|FocusTrapZones.Panel on its own.default.chromium.png|0| Added|
|FolderCover.Large Default Cover.default.chromium.png|0| Added|
|FolderCover.Large Media Cover.default.chromium.png|0| Added|
|FolderCover.Shared Cover.default.chromium.png|0| Added|
|FolderCover.Small Default Cover.default.chromium.png|0| Added|
|FolderCover.Small Media Cover.default.chromium.png|0| Added|
|Fonts.Arabic.default.chromium.png|0| Added|
|Fonts.Armenian.default.chromium.png|0| Added|
|Fonts.Chinese (Simplified).default.chromium.png|0| Added|
|Fonts.Chinese (Traditional).default.chromium.png|0| Added|
|Fonts.Cyrillic.default.chromium.png|0| Added|
|Fonts.East European.default.chromium.png|0| Added|
|Fonts.Georgian.default.chromium.png|0| Added|
|Fonts.Greek.default.chromium.png|0| Added|
|Fonts.Hebrew.default.chromium.png|0| Added|
|Fonts.Hindi.default.chromium.png|0| Added|
|Fonts.Japanese.default.chromium.png|0| Added|
|Fonts.Korean.default.chromium.png|0| Added|
|Fonts.Thai.default.chromium.png|0| Added|
|Fonts.Vietnamese.default.chromium.png|0| Added|
|Fonts.Weights.default.chromium.png|0| Added|
|Fonts.West European.default.chromium.png|0| Added|
|GroupedList.Root - RTL.click.chromium.png|0| Added|
|GroupedList.Root - RTL.default.chromium.png|0| Added|
|GroupedList.Root - RTL.hover.chromium.png|0| Added|
|GroupedList.Root.click.chromium.png|0| Added|
|GroupedList.Root.default.chromium.png|0| Added|
|GroupedList.Root.hover.chromium.png|0| Added|
|HoverCard.Root - RTL.default.chromium.png|0| Added|
|HoverCard.Root - RTL.fully expanded with test content.chromium.png|0| Added|
|HoverCard.Root.default.chromium.png|0| Added|
|HoverCard.Root.fully expanded with test content.chromium.png|0| Added|
|Icon.Color.default.chromium.png|0| Added|
|Icon.Image.default.chromium.png|0| Added|
|Icon.Root.default.chromium.png|0| Added|
|IconButton Scenarios (compat).icon button with menu.hover icon.chromium.png|0| Added|
|IconButton Scenarios (compat).icon button with menu.icon.chromium.png|0| Added|
|IconButton Scenarios (compat).normal icon button.hover icon.chromium.png|0| Added|
|IconButton Scenarios (compat).normal icon button.icon.chromium.png|0| Added|
|Image.Fit- CenterContain, image larger.default.chromium.png|0| Added|
|Image.Fit- CenterContain, image smaller.default.chromium.png|0| Added|
|Image.Fit- CenterContain, image taller.default.chromium.png|0| Added|
|Image.Fit- CenterContain, image wider.default.chromium.png|0| Added|
|Image.Fit- center, image larger.default.chromium.png|0| Added|
|Image.Fit- center, image smaller.default.chromium.png|0| Added|
|Image.Fit- centerCover, image larger.default.chromium.png|0| Added|
|Image.Fit- centerCover, image smaller.default.chromium.png|0| Added|
|Image.Fit- centerCover, image taller.default.chromium.png|0| Added|
|Image.Fit- centerCover, image wider.default.chromium.png|0| Added|
|Image.Fit- contain, image taller.default.chromium.png|0| Added|
|Image.Fit- contain, image wider.default.chromium.png|0| Added|
|Image.Fit- cover, image taller.default.chromium.png|0| Added|
|Image.Fit- cover, image wider.default.chromium.png|0| Added|
|Image.Fit- none, image larger.default.chromium.png|0| Added|
|Image.Fit- none, image smaller.default.chromium.png|0| Added|
|Image.Maximize frame, landscape container.default.chromium.png|0| Added|
|Image.Maximize frame, portrait container.default.chromium.png|0| Added|
|Image.No fit, no w-h.default.chromium.png|0| Added|
|Image.No fit, only height.default.chromium.png|0| Added|
|Image.No fit, only width.default.chromium.png|0| Added|
|Keytip.Disabled.default.chromium.png|0| Added|
|Keytip.Offset.default.chromium.png|0| Added|
|Keytip.Root.default.chromium.png|0| Added|
|Label.Disabled.default.chromium.png|0| Added|
|Label.Required.default.chromium.png|0| Added|
|Label.Root - RTL.default.chromium.png|0| Added|
|Label.Root.default.chromium.png|0| Added|
|Layer.Root - RTL.default.chromium.png|0| Added|
|Layer.Root.default.chromium.png|0| Added|
|Link.Disabled.click.chromium.png|0| Added|
|Link.Disabled.default.chromium.png|0| Added|
|Link.Disabled.focus.chromium.png|0| Added|
|Link.Disabled.hover.chromium.png|0| Added|
|Link.No Href Disabled.click.chromium.png|0| Added|
|Link.No Href Disabled.default.chromium.png|0| Added|
|Link.No Href Disabled.focus.chromium.png|0| Added|
|Link.No Href Disabled.hover.chromium.png|0| Added|
|Link.No Href.click.chromium.png|0| Added|
|Link.No Href.default.chromium.png|0| Added|
|Link.No Href.focus.chromium.png|0| Added|
|Link.No Href.hover.chromium.png|0| Added|
|Link.Root - RTL.click.chromium.png|0| Added|
|Link.Root - RTL.default.chromium.png|0| Added|
|Link.Root - RTL.focus.chromium.png|0| Added|
|Link.Root - RTL.hover.chromium.png|0| Added|
|Link.Root.click.chromium.png|0| Added|
|Link.Root.default.chromium.png|0| Added|
|Link.Root.focus.chromium.png|0| Added|
|Link.Root.hover.chromium.png|0| Added|
|Link.Underlined Disabled.click.chromium.png|0| Added|
|Link.Underlined Disabled.default.chromium.png|0| Added|
|Link.Underlined Disabled.focus.chromium.png|0| Added|
|Link.Underlined Disabled.hover.chromium.png|0| Added|
|Link.Underlined.click.chromium.png|0| Added|
|Link.Underlined.default.chromium.png|0| Added|
|Link.Underlined.focus.chromium.png|0| Added|
|Link.Underlined.hover.chromium.png|0| Added|
|List.Root - RTL.default.chromium.png|0| Added|
|List.Root.default.chromium.png|0| Added|
|MediaTile.Media tile with nameplate hidden until hover.default.chromium.png|0| Added|
|MediaTile.Media tile with nameplate hidden until hover.hover.chromium.png|0| Added|
|MediaTile.Media tile with single activity line.default.chromium.png|0| Added|
|MediaTile.Media tile with single activity line.hover.chromium.png|0| Added|
|MediaTile.Media tile with two activity lines.default.chromium.png|0| Added|
|MediaTile.Media tile with two activity lines.hover.chromium.png|0| Added|
|MediaTile.Media tile with very long name and activity.default.chromium.png|0| Added|
|MediaTile.Media tile with very long name and activity.hover.chromium.png|0| Added|
|MessageBar.Blocked.default.chromium.png|0| Added|
|MessageBar.Error.default.chromium.png|0| Added|
|MessageBar.Root - RTL.default.chromium.png|0| Added|
|MessageBar.Root actions - RTL.default.chromium.png|0| Added|
|MessageBar.Root actions single line - RTL.default.chromium.png|0| Added|
|MessageBar.Root actions single line.default.chromium.png|0| Added|
|MessageBar.Root actions.default.chromium.png|0| Added|
|MessageBar.Root dismiss - RTL.default.chromium.png|0| Added|
|MessageBar.Root dismiss and action - RTL.default.chromium.png|0| Added|
|MessageBar.Root dismiss and action single line - RTL.default.chromium.png|0| Added|
|MessageBar.Root dismiss and action single line.default.chromium.png|0| Added|
|MessageBar.Root dismiss and action.default.chromium.png|0| Added|
|MessageBar.Root dismiss single line.default.chromium.png|0| Added|
|MessageBar.Root dismiss.default.chromium.png|0| Added|
|MessageBar.Root multiline - RTL.default.chromium.png|0| Added|
|MessageBar.Root multiline.default.chromium.png|0| Added|
|MessageBar.Root overflow - RTL.default.chromium.png|0| Added|
|MessageBar.Root overflow.default.chromium.png|0| Added|
|MessageBar.Root truncated - RTL.default.chromium.png|0| Added|
|MessageBar.Root truncated.default.chromium.png|0| Added|
|MessageBar.Root.default.chromium.png|0| Added|
|MessageBar.Severe Warning.default.chromium.png|0| Added|
|MessageBar.Success.default.chromium.png|0| Added|
|MessageBar.Warning.default.chromium.png|0| Added|
|Modal.Modeless.default.chromium.png|0| Added|
|Modal.Root.default.chromium.png|0| Added|
|Nav.Disabled - RTL.click.chromium.png|0| Added|
|Nav.Disabled - RTL.default.chromium.png|0| Added|
|Nav.Disabled - RTL.hover.chromium.png|0| Added|
|Nav.Disabled.click.chromium.png|0| Added|
|Nav.Disabled.default.chromium.png|0| Added|
|Nav.Disabled.hover.chromium.png|0| Added|
|Nav.Root - RTL.click.chromium.png|0| Added|
|Nav.Root - RTL.default.chromium.png|0| Added|
|Nav.Root - RTL.hover.chromium.png|0| Added|
|Nav.Root.click.chromium.png|0| Added|
|Nav.Root.default.chromium.png|0| Added|
|Nav.Root.hover.chromium.png|0| Added|
|OverflowSet variant.Vertical Direction.default.chromium.png|0| Added|
|OverflowSet.Root - RTL.default.chromium.png|0| Added|
|OverflowSet.Root.default.chromium.png|0| Added|
|Overlay.Dark.default.chromium.png|0| Added|
|Overlay.Root - RTL.default.chromium.png|0| Added|
|Overlay.Root.default.chromium.png|0| Added|
|Panel.Custom anchored left.default.chromium.png|0| Added|
|Panel.Custom.default.chromium.png|0| Added|
|Panel.Extra large right.default.chromium.png|0| Added|
|Panel.Large fixed right.default.chromium.png|0| Added|
|Panel.Large right.default.chromium.png|0| Added|
|Panel.Medium right - RTL.default.chromium.png|0| Added|
|Panel.Medium right.default.chromium.png|0| Added|
|Panel.SearchBox and Right Panel - RTL.click.chromium.png|0| Added|
|Panel.SearchBox and Right Panel - RTL.default.chromium.png|0| Added|
|Panel.SearchBox and Right Panel.click.chromium.png|0| Added|
|Panel.SearchBox and Right Panel.default.chromium.png|0| Added|
|Panel.Small fixed right w- close button - RTL.default.chromium.png|0| Added|
|Panel.Small fixed right w- close button.default.chromium.png|0| Added|
|Panel.Small fluid right.default.chromium.png|0| Added|
|Panel.Small left w- close button - RTL.default.chromium.png|0| Added|
|Panel.Small left w- close button.default.chromium.png|0| Added|
|Panel.With custom navigation.default.chromium.png|0| Added|
|Panel.With footer at the bottom.default.chromium.png|0| Added|
|Panel.With no header, close button.default.chromium.png|0| Added|
|Panel.With no navigation.default.chromium.png|0| Added|
|PeoplePicker.Compact disabled.chromium.png|0| Added|
|PeoplePicker.Compact selected - RTL.default.chromium.png|0| Added|
|PeoplePicker.Compact selected - RTL.suggestions.chromium.png|0| Added|
|PeoplePicker.Compact selected.default.chromium.png|0| Added|
|PeoplePicker.Compact selected.suggestions.chromium.png|0| Added|
|PeoplePicker.Compact.default.chromium.png|0| Added|
|PeoplePicker.Compact.suggestions.chromium.png|0| Added|
|PeoplePicker.List disabled.chromium.png|0| Added|
|PeoplePicker.List selected - RTL.default.chromium.png|0| Added|
|PeoplePicker.List selected - RTL.suggestions.chromium.png|0| Added|
|PeoplePicker.List selected.default.chromium.png|0| Added|
|PeoplePicker.List selected.suggestions.chromium.png|0| Added|
|PeoplePicker.List.default.chromium.png|0| Added|
|PeoplePicker.List.suggestions.chromium.png|0| Added|
|PeoplePicker.Normal Overflow selected - RTL.default.chromium.png|0| Added|
|PeoplePicker.Normal Overflow selected - RTL.suggestions.chromium.png|0| Added|
|PeoplePicker.Normal Overflow selected.default.chromium.png|0| Added|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|0| Added|
|PeoplePicker.Normal disabled with placeholder.chromium.png|0| Added|
|PeoplePicker.Normal disabled.chromium.png|0| Added|
|PeoplePicker.Normal selected - RTL.default.chromium.png|0| Added|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|0| Added|
|PeoplePicker.Normal selected.default.chromium.png|0| Added|
|PeoplePicker.Normal selected.suggestions.chromium.png|0| Added|
|PeoplePicker.Normal with placeholder.chromium.png|0| Added|
|PeoplePicker.Normal with text.default.chromium.png|0| Added|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|0| Added|
|PeoplePicker.Normal.default.chromium.png|0| Added|
|PeoplePicker.Normal.suggestions.chromium.png|0| Added|
|Persona.Initials - RTL.default.chromium.png|0| Added|
|Persona.Initials.default.chromium.png|0| Added|
|Persona.Persona with children.default.chromium.png|0| Added|
|Persona.default size, details hidden.default.chromium.png|0| Added|
|Persona.default size, presences.default.chromium.png|0| Added|
|Persona.size10 (tiny).default.chromium.png|0| Added|
|Persona.size100 (extraLarge) - RTL.default.chromium.png|0| Added|
|Persona.size100 (extraLarge).default.chromium.png|0| Added|
|Persona.size120 - RTL.default.chromium.png|0| Added|
|Persona.size120.default.chromium.png|0| Added|
|Persona.size24 (extraExtraSmall).default.chromium.png|0| Added|
|Persona.size28 (extraSmall).default.chromium.png|0| Added|
|Persona.size32.default.chromium.png|0| Added|
|Persona.size40 (small).default.chromium.png|0| Added|
|Persona.size48 (regular).default.chromium.png|0| Added|
|Persona.size72 (large).default.chromium.png|0| Added|
|Pivot - Overflow.Root.Medium.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Root.Wide.chromium.png|0| Added|
|Pivot - Overflow.Root.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Medium.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Wide.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.chromium.png|0| Added|
|Pivot.Count and icon - RTL.click.chromium.png|0| Added|
|Pivot.Count and icon - RTL.default.chromium.png|0| Added|
|Pivot.Count and icon - RTL.hover-selected.chromium.png|0| Added|
|Pivot.Count and icon - RTL.hover.chromium.png|0| Added|
|Pivot.Count and icon.click.chromium.png|0| Added|
|Pivot.Count and icon.default.chromium.png|0| Added|
|Pivot.Count and icon.hover-selected.chromium.png|0| Added|
|Pivot.Count and icon.hover.chromium.png|0| Added|
|Pivot.Large.click.chromium.png|0| Added|
|Pivot.Large.default.chromium.png|0| Added|
|Pivot.Large.hover-selected.chromium.png|0| Added|
|Pivot.Large.hover.chromium.png|0| Added|
|Pivot.Root.click.chromium.png|0| Added|
|Pivot.Root.default.chromium.png|0| Added|
|Pivot.Root.hover-selected.chromium.png|0| Added|
|Pivot.Root.hover.chromium.png|0| Added|
|Pivot.Tabs - RTL.click.chromium.png|0| Added|
|Pivot.Tabs - RTL.default.chromium.png|0| Added|
|Pivot.Tabs - RTL.hover-selected.chromium.png|0| Added|
|Pivot.Tabs - RTL.hover.chromium.png|0| Added|
|Pivot.Tabs large.click.chromium.png|0| Added|
|Pivot.Tabs large.default.chromium.png|0| Added|
|Pivot.Tabs large.hover-selected.chromium.png|0| Added|
|Pivot.Tabs large.hover.chromium.png|0| Added|
|Pivot.Tabs.click.chromium.png|0| Added|
|Pivot.Tabs.default.chromium.png|0| Added|
|Pivot.Tabs.hover-selected.chromium.png|0| Added|
|Pivot.Tabs.hover.chromium.png|0| Added|
|ProgressIndicator.0%.default.chromium.png|0| Added|
|ProgressIndicator.100%.default.chromium.png|0| Added|
|ProgressIndicator.50% - RTL.default.chromium.png|0| Added|
|ProgressIndicator.50%.default.chromium.png|0| Added|
|Rating.Disabled.click.chromium.png|0| Added|
|Rating.Disabled.default.chromium.png|0| Added|
|Rating.Large.click.chromium.png|0| Added|
|Rating.Large.default.chromium.png|0| Added|
|Rating.Rated - RTL.click.chromium.png|0| Added|
|Rating.Rated - RTL.default.chromium.png|0| Added|
|Rating.Rated.click.chromium.png|0| Added|
|Rating.Rated.default.chromium.png|0| Added|
|Rating.Root.click.chromium.png|0| Added|
|Rating.Root.default.chromium.png|0| Added|
|ResizeGroup.Root - RTL.click overflow.chromium.png|0| Added|
|ResizeGroup.Root - RTL.default.chromium.png|0| Added|
|ResizeGroup.Root.click overflow.chromium.png|0| Added|
|ResizeGroup.Root.default.chromium.png|0| Added|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.default.chromium.png|0| Added|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll down by a small amount so that the first row is still visible.chromium.png|0| Added|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll down to the bottom.chromium.png|0| Added|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll up to the top.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.If groups are expanded, on horizontal scroll, over scroll should not happen for content container.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.If groups are expanded, when header becomes sticky, horizontal scrollbar should be visible.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.On expanding a group, horizontal scrollbar should be visible.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling down to the bottom with non-zero scrollLeft.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling down to the bottom.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling up to the top with non-zero scrollLeft.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling up to the top.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility when header is sticky and scrollLeft is non-zero.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility when header is sticky.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibilty after scrolling left.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.default- scrollbars should be visible.chromium.png|0| Added|
|ScrollablePane.Default ScrollablePane Example.default.chromium.png|0| Added|
|ScrollablePane.Default ScrollablePane Example.scrolled.chromium.png|0| Added|
|SearchBox.Full - RTL.default.chromium.png|0| Added|
|SearchBox.Full.default.chromium.png|0| Added|
|SearchBox.Root - RTL.default.chromium.png|0| Added|
|SearchBox.Root.default.chromium.png|0| Added|
|SearchBox.ShowIcon - RTL.default.chromium.png|0| Added|
|SearchBox.ShowIcon.default.chromium.png|0| Added|
|Separator.Horizontal End Aligned - RTL.default.chromium.png|0| Added|
|Separator.Horizontal End Aligned.default.chromium.png|0| Added|
|Separator.Horizontal Start Aligned - RTL.default.chromium.png|0| Added|
|Separator.Horizontal Start Aligned.default.chromium.png|0| Added|
|Separator.Root - RTL.default.chromium.png|0| Added|
|Separator.Root.default.chromium.png|0| Added|
|Separator.Vertical Center Aligned - RTL.default.chromium.png|0| Added|
|Separator.Vertical Center Aligned.default.chromium.png|0| Added|
|Separator.Vertical End Aligned - RTL.default.chromium.png|0| Added|
|Separator.Vertical End Aligned.default.chromium.png|0| Added|
|Separator.Vertical Start Aligned - RTL.default.chromium.png|0| Added|
|Separator.Vertical Start Aligned.default.chromium.png|0| Added|
|Shimmer.50% width.default.chromium.png|0| Added|
|Shimmer.Basic.default.chromium.png|0| Added|
|Shimmer.Circle Gap Line - RTL.default.chromium.png|0| Added|
|Shimmer.Circle Gap Line.default.chromium.png|0| Added|
|Shimmer.Custom elements on themed background.default.chromium.png|0| Added|
|Shimmer.Custom elements.default.chromium.png|0| Added|
|Shimmer.Data loaded.default.chromium.png|0| Added|
|Shimmer.Data not loaded.default.chromium.png|0| Added|
|Signals.Awaiting approval.default.chromium.png|0| Added|
|Signals.Blocked.default.chromium.png|0| Added|
|Signals.Comments (count).default.chromium.png|0| Added|
|Signals.Comments.default.chromium.png|0| Added|
|Signals.Emailed.default.chromium.png|0| Added|
|Signals.Follow.default.chromium.png|0| Added|
|Signals.Malware detected.default.chromium.png|0| Added|
|Signals.Mention.default.chromium.png|0| Added|
|Signals.Missing metadata.default.chromium.png|0| Added|
|Signals.New (positioning).default.chromium.png|0| Added|
|Signals.New.default.chromium.png|0| Added|
|Signals.NotFollow.default.chromium.png|0| Added|
|Signals.Read-only.default.chromium.png|0| Added|
|Signals.Record.default.chromium.png|0| Added|
|Signals.Shared.default.chromium.png|0| Added|
|Signals.Someone checked out.default.chromium.png|0| Added|
|Signals.Trending.default.chromium.png|0| Added|
|Signals.Unseen edit.default.chromium.png|0| Added|
|Signals.Unseen reply.default.chromium.png|0| Added|
|Signals.Warning.default.chromium.png|0| Added|
|Signals.You checked out.default.chromium.png|0| Added|
|Slider.Disabled.default.chromium.png|0| Added|
|Slider.Disabled.hover.chromium.png|0| Added|
|Slider.EqualMinMax.default.chromium.png|0| Added|
|Slider.EqualMinMax.hover.chromium.png|0| Added|
|Slider.Max not multiple of step.default.chromium.png|0| Added|
|Slider.Max not multiple of step.hover.chromium.png|0| Added|
|Slider.Root - RTL.default.chromium.png|0| Added|
|Slider.Root - RTL.hover.chromium.png|0| Added|
|Slider.Root.default.chromium.png|0| Added|
|Slider.Root.hover.chromium.png|0| Added|
|Slider.Step less than 1.default.chromium.png|0| Added|
|Slider.Step less than 1.hover.chromium.png|0| Added|
|Slider.Vertical.default.chromium.png|0| Added|
|Slider.Vertical.hover.chromium.png|0| Added|
|SpinButton.Disabled.click input.chromium.png|0| Added|
|SpinButton.Disabled.default.chromium.png|0| Added|
|SpinButton.Disabled.hover arrow.chromium.png|0| Added|
|SpinButton.Disabled.hover input.chromium.png|0| Added|
|SpinButton.Disabled.mouseDown arrow.chromium.png|0| Added|
|SpinButton.Label at end - RTL.default.chromium.png|0| Added|
|SpinButton.Label at end with icon - RTL.default.chromium.png|0| Added|
|SpinButton.Label at end with icon.default.chromium.png|0| Added|
|SpinButton.Label at end.default.chromium.png|0| Added|
|SpinButton.Label on bottom with icon.default.chromium.png|0| Added|
|SpinButton.Label on bottom.default.chromium.png|0| Added|
|SpinButton.Label on top with icon.default.chromium.png|0| Added|
|SpinButton.Label on top.default.chromium.png|0| Added|
|SpinButton.Root.click input.chromium.png|0| Added|
|SpinButton.Root.default.chromium.png|0| Added|
|SpinButton.Root.hover arrow.chromium.png|0| Added|
|SpinButton.Root.hover input.chromium.png|0| Added|
|SpinButton.Root.mouseDown arrow.chromium.png|0| Added|
|SpinButton.With icon - RTL.click input.chromium.png|0| Added|
|SpinButton.With icon - RTL.default.chromium.png|0| Added|
|SpinButton.With icon - RTL.hover arrow.chromium.png|0| Added|
|SpinButton.With icon - RTL.hover input.chromium.png|0| Added|
|SpinButton.With icon - RTL.mouseDown arrow.chromium.png|0| Added|
|SpinButton.With icon.click input.chromium.png|0| Added|
|SpinButton.With icon.default.chromium.png|0| Added|
|SpinButton.With icon.hover arrow.chromium.png|0| Added|
|SpinButton.With icon.hover input.chromium.png|0| Added|
|SpinButton.With icon.mouseDown arrow.chromium.png|0| Added|
|Spinner.Extra small.default.chromium.png|0| Added|
|Spinner.Label at top.default.chromium.png|0| Added|
|Spinner.Label on the left.default.chromium.png|0| Added|
|Spinner.Label on the right.default.chromium.png|0| Added|
|Spinner.Label.default.chromium.png|0| Added|
|Spinner.Large.default.chromium.png|0| Added|
|Spinner.Medium.default.chromium.png|0| Added|
|Spinner.Small.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Bottom-aligned.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Box shadow around items.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Default - RTL.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Default.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Gap - RTL.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Gap.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Growing items.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Horizontally centered.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Item alignments - RTL.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Item alignments.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Padding.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Reversed - RTL.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Reversed.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Right-aligned - RTL.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Right-aligned.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Shrinking items.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Space around.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Space between.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Space evenly.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Vertically centered.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Wrap - RTL.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Wrap with specified vertical gap.default.chromium.png|0| Added|
|Stack.Horizontal Stack - Wrap.default.chromium.png|0| Added|
|Stack.Vertical Stack - Bottom-aligned.default.chromium.png|0| Added|
|Stack.Vertical Stack - Box shadow around items.default.chromium.png|0| Added|
|Stack.Vertical Stack - Default - RTL.default.chromium.png|0| Added|
|Stack.Vertical Stack - Default.default.chromium.png|0| Added|
|Stack.Vertical Stack - Gap.default.chromium.png|0| Added|
|Stack.Vertical Stack - Growing items.default.chromium.png|0| Added|
|Stack.Vertical Stack - Horizontally centered.default.chromium.png|0| Added|
|Stack.Vertical Stack - Item alignments - RTL.default.chromium.png|0| Added|
|Stack.Vertical Stack - Item alignments.default.chromium.png|0| Added|
|Stack.Vertical Stack - Padding.default.chromium.png|0| Added|
|Stack.Vertical Stack - Reversed - RTL.default.chromium.png|0| Added|
|Stack.Vertical Stack - Reversed.default.chromium.png|0| Added|
|Stack.Vertical Stack - Right-aligned - RTL.default.chromium.png|0| Added|
|Stack.Vertical Stack - Right-aligned.default.chromium.png|0| Added|
|Stack.Vertical Stack - Shrinking items.default.chromium.png|0| Added|
|Stack.Vertical Stack - Space around.default.chromium.png|0| Added|
|Stack.Vertical Stack - Space between.default.chromium.png|0| Added|
|Stack.Vertical Stack - Space evenly.default.chromium.png|0| Added|
|Stack.Vertical Stack - Vertically centered.default.chromium.png|0| Added|
|Stack.Vertical Stack - Wrap - RTL.default.chromium.png|0| Added|
|Stack.Vertical Stack - Wrap.default.chromium.png|0| Added|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.default.chromium.png|0| Added|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll back to the top when horizontal scroll is non-zero.chromium.png|0| Added|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down by a small amount so that the first row is still visible and the header becomes sticky.chromium.png|0| Added|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down to the bottom.chromium.png|0| Added|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down when horizontal scroll is non-zero.chromium.png|0| Added|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll right.chromium.png|0| Added|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll up to the top.chromium.png|0| Added|
|SwatchColorPicker.Circle - RTL.Focused.chromium.png|0| Added|
|SwatchColorPicker.Circle - RTL.Hovered.chromium.png|0| Added|
|SwatchColorPicker.Circle - RTL.Mousedown.chromium.png|0| Added|
|SwatchColorPicker.Circle - RTL.Selected and Focused.chromium.png|0| Added|
|SwatchColorPicker.Circle - RTL.Selected and Hovered.chromium.png|0| Added|
|SwatchColorPicker.Circle - RTL.default.chromium.png|0| Added|
|SwatchColorPicker.Circle over 24px size.Focused.chromium.png|0| Added|
|SwatchColorPicker.Circle over 24px size.Hovered.chromium.png|0| Added|
|SwatchColorPicker.Circle over 24px size.Mousedown.chromium.png|0| Added|
|SwatchColorPicker.Circle over 24px size.Selected and Focused.chromium.png|0| Added|
|SwatchColorPicker.Circle over 24px size.Selected and Hovered.chromium.png|0| Added|
|SwatchColorPicker.Circle over 24px size.default.chromium.png|0| Added|
|SwatchColorPicker.Circle.Focused.chromium.png|0| Added|
|SwatchColorPicker.Circle.Hovered.chromium.png|0| Added|
|SwatchColorPicker.Circle.Mousedown.chromium.png|0| Added|
|SwatchColorPicker.Circle.Selected and Focused.chromium.png|0| Added|
|SwatchColorPicker.Circle.Selected and Hovered.chromium.png|0| Added|
|SwatchColorPicker.Circle.default.chromium.png|0| Added|
|SwatchColorPicker.Disabled.Focused.chromium.png|0| Added|
|SwatchColorPicker.Disabled.Hovered.chromium.png|0| Added|
|SwatchColorPicker.Disabled.Mousedown.chromium.png|0| Added|
|SwatchColorPicker.Disabled.Selected and Focused.chromium.png|0| Added|
|SwatchColorPicker.Disabled.Selected and Hovered.chromium.png|0| Added|
|SwatchColorPicker.Disabled.default.chromium.png|0| Added|
|SwatchColorPicker.Multiple rows.Focused.chromium.png|0| Added|
|SwatchColorPicker.Multiple rows.Hovered.chromium.png|0| Added|
|SwatchColorPicker.Multiple rows.Mousedown.chromium.png|0| Added|
|SwatchColorPicker.Multiple rows.Selected and Focused.chromium.png|0| Added|
|SwatchColorPicker.Multiple rows.Selected and Hovered.chromium.png|0| Added|
|SwatchColorPicker.Multiple rows.default.chromium.png|0| Added|
|SwatchColorPicker.Square over 24px size.Focused.chromium.png|0| Added|
|SwatchColorPicker.Square over 24px size.Hovered.chromium.png|0| Added|
|SwatchColorPicker.Square over 24px size.Mousedown.chromium.png|0| Added|
|SwatchColorPicker.Square over 24px size.Selected and Focused.chromium.png|0| Added|
|SwatchColorPicker.Square over 24px size.Selected and Hovered.chromium.png|0| Added|
|SwatchColorPicker.Square over 24px size.default.chromium.png|0| Added|
|SwatchColorPicker.Square.Focused.chromium.png|0| Added|
|SwatchColorPicker.Square.Hovered.chromium.png|0| Added|
|SwatchColorPicker.Square.Mousedown.chromium.png|0| Added|
|SwatchColorPicker.Square.Selected and Focused.chromium.png|0| Added|
|SwatchColorPicker.Square.Selected and Hovered.chromium.png|0| Added|
|SwatchColorPicker.Square.default.chromium.png|0| Added|
|TagItem.Selected - RTL.Tag Item Clear Button Hover.chromium.png|0| Added|
|TagItem.Selected - RTL.Tag Item Clear Button Selected Hover.chromium.png|0| Added|
|TagItem.Selected - RTL.Tag Item Hover.chromium.png|0| Added|
|TagItem.Selected - RTL.Tag Item Select.chromium.png|0| Added|
|TagItem.Selected - RTL.default.chromium.png|0| Added|
|TagItem.Selected.Tag Item Clear Button Hover.chromium.png|0| Added|
|TagItem.Selected.Tag Item Clear Button Selected Hover.chromium.png|0| Added|
|TagItem.Selected.Tag Item Hover.chromium.png|0| Added|
|TagItem.Selected.Tag Item Select.chromium.png|0| Added|
|TagItem.Selected.default.chromium.png|0| Added|
|TagPicker.Root.Open Suggestion Menu.chromium.png|0| Added|
|TagPicker.Root.Suggestion Menu Item Hover.chromium.png|0| Added|
|TagPicker.Root.default.chromium.png|0| Added|
|TagPicker.Selected - RTL.Open Suggestion Menu.chromium.png|0| Added|
|TagPicker.Selected - RTL.Suggestion Menu Item Hover.chromium.png|0| Added|
|TagPicker.Selected - RTL.default.chromium.png|0| Added|
|TagPicker.Selected.Open Suggestion Menu.chromium.png|0| Added|
|TagPicker.Selected.Suggestion Menu Item Hover.chromium.png|0| Added|
|TagPicker.Selected.default.chromium.png|0| Added|
|TagPicker.TagPicker disabled.chromium.png|0| Added|
|TagPicker.With long tag.default.chromium.png|0| Added|
|TeachingBubble.SmallHeadline.default.chromium.png|0| Added|
|TeachingBubble.WideIllustration - RTL.default.chromium.png|0| Added|
|TeachingBubble.WideIllustration.default.chromium.png|0| Added|
|Text.Block.default.chromium.png|0| Added|
|Text.No wrap.default.chromium.png|0| Added|
|Text.Root - RTL.default.chromium.png|0| Added|
|Text.Root.default.chromium.png|0| Added|
|Text.Wrap.default.chromium.png|0| Added|
|TextField.Borderless.click.chromium.png|0| Added|
|TextField.Borderless.default.chromium.png|0| Added|
|TextField.Borderless.hover.chromium.png|0| Added|
|TextField.Disabled.click.chromium.png|0| Added|
|TextField.Disabled.default.chromium.png|0| Added|
|TextField.Disabled.hover.chromium.png|0| Added|
|TextField.Error - RTL.click.chromium.png|0| Added|
|TextField.Error - RTL.default.chromium.png|0| Added|
|TextField.Error - RTL.hover.chromium.png|0| Added|
|TextField.Error.click.chromium.png|0| Added|
|TextField.Error.default.chromium.png|0| Added|
|TextField.Error.hover.chromium.png|0| Added|
|TextField.Icon - RTL.click.chromium.png|0| Added|
|TextField.Icon - RTL.default.chromium.png|0| Added|
|TextField.Icon - RTL.hover.chromium.png|0| Added|
|TextField.Icon.click.chromium.png|0| Added|
|TextField.Icon.default.chromium.png|0| Added|
|TextField.Icon.hover.chromium.png|0| Added|
|TextField.Multiline - RTL.click.chromium.png|0| Added|
|TextField.Multiline - RTL.default.chromium.png|0| Added|
|TextField.Multiline - RTL.hover.chromium.png|0| Added|
|TextField.Multiline nonresizable.click.chromium.png|0| Added|
|TextField.Multiline nonresizable.default.chromium.png|0| Added|
|TextField.Multiline nonresizable.hover.chromium.png|0| Added|
|TextField.Multiline.click.chromium.png|0| Added|
|TextField.Multiline.default.chromium.png|0| Added|
|TextField.Multiline.hover.chromium.png|0| Added|
|TextField.Placeholder - RTL.click.chromium.png|0| Added|
|TextField.Placeholder - RTL.default.chromium.png|0| Added|
|TextField.Placeholder - RTL.hover.chromium.png|0| Added|
|TextField.Placeholder.click.chromium.png|0| Added|
|TextField.Placeholder.default.chromium.png|0| Added|
|TextField.Placeholder.hover.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled - RTL.click.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled - RTL.default.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled - RTL.hover.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.click.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.default.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.hover.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.click.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.default.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.hover.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled.click.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled.default.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled.hover.chromium.png|0| Added|
|TextField.Required.click.chromium.png|0| Added|
|TextField.Required.default.chromium.png|0| Added|
|TextField.Required.hover.chromium.png|0| Added|
|TextField.Root.click.chromium.png|0| Added|
|TextField.Root.default.chromium.png|0| Added|
|TextField.Root.hover.chromium.png|0| Added|
|TextField.Suffix - RTL.click.chromium.png|0| Added|
|TextField.Suffix - RTL.default.chromium.png|0| Added|
|TextField.Suffix - RTL.hover.chromium.png|0| Added|
|TextField.Suffix.click.chromium.png|0| Added|
|TextField.Suffix.default.chromium.png|0| Added|
|TextField.Suffix.hover.chromium.png|0| Added|
|TextField.Underlined.click.chromium.png|0| Added|
|TextField.Underlined.default.chromium.png|0| Added|
|TextField.Underlined.hover.chromium.png|0| Added|
|ThemeProvider with Customizer.Customizer wraps ThemeProvider.default.chromium.png|0| Added|
|ThemeProvider with Customizer.ThemeProvider wraps Customizer.default.chromium.png|0| Added|
|ThemeProvider with loadTheme.Use contextual theme over global theme if defined.default.chromium.png|0| Added|
|ThemeProvider with loadTheme.Use contextual theme over global theme if defined.global theme changed.chromium.png|0| Added|
|ThemeProvider with loadTheme.Use updated global theme.default.chromium.png|0| Added|
|ThemeProvider with loadTheme.Use updated global theme.global theme changed.chromium.png|0| Added|
|ThemeProvider.Customized styles.default.chromium.png|0| Added|
|ThemeProvider.Customized theme - nested ThemeProvider.default.chromium.png|0| Added|
|ThemeProvider.Customized theme.default.chromium.png|0| Added|
|ThemeProvider.Default theme.default.chromium.png|0| Added|
|Tile.Document tile with fit landscape image.default.chromium.png|0| Added|
|Tile.Document tile with fit portrait image.default.chromium.png|0| Added|
|Tile.Document tile with icon-sized image.default.chromium.png|0| Added|
|Tile.Document tile with icon.default.chromium.png|0| Added|
|Tile.Tile with no content and long text.default.chromium.png|0| Added|
|Toggle.Checked - RTL.default.chromium.png|0| Added|
|Toggle.Checked.default.chromium.png|0| Added|
|Toggle.Disabled checked.default.chromium.png|0| Added|
|Toggle.Disabled unchecked.default.chromium.png|0| Added|
|Toggle.Unchecked - RTL.default.chromium.png|0| Added|
|Toggle.Unchecked.default.chromium.png|0| Added|
|Toggle.With inline label (JSX Element) and without onText and offText.default.chromium.png|0| Added|
|Toggle.With inline label (JSX Element).default.chromium.png|0| Added|
|Toggle.With inline label and without onText and offText.default.chromium.png|0| Added|
|Toggle.With inline label disabled and without onText and offText.default.chromium.png|0| Added|
|Toggle.With inline label disabled.default.chromium.png|0| Added|
|Toggle.With inline label.default.chromium.png|0| Added|
|Tooltip - Multiple.Two Tooltips.chromium.png|0| Added|
|Tooltip - Multiple.Two Tooltips.hover inner.chromium.png|0| Added|
|Tooltip - Multiple.Two Tooltips.hover outer.chromium.png|0| Added|
|Tooltip.Default.chromium.png|0| Added|
|Tooltip.Default.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>